diff --git a/src/app/components/Board.js b/src/app/components/Board.js
index ef8bbe8..9d544e0 100644
--- a/src/app/components/Board.js
+++ b/src/app/components/Board.js
@@ -1,8 +1,8 @@
 "use client";
-import { useState, useRef } from "react";
-import Dragula from "dragula";
+import { useState, useRef, useEffect } from "react";
 import "dragula/dist/dragula.css";
 import Swimlane from "./Swimlane";
+import dragula from "dragula";
 
 function getClients() {
   return [
@@ -80,7 +80,7 @@ function getClients() {
     id: companyDetails[0],
     name: companyDetails[1],
     description: companyDetails[2],
-    status: companyDetails[3],
+    status: "backlog",
   }));
 }
 
@@ -96,6 +96,35 @@ export default function Board() {
     return <Swimlane name={name} clients={clients} dragulaRef={ref} />;
   }
 
+  useEffect(() => {
+    const drake = dragula([
+      swimlanes.backlog.current,
+      swimlanes.inProgress.current,
+      swimlanes.complete.current,
+    ]);
+
+    drake.on("drop", (el, target, source, sibling) => {
+      // console.log("el", el.datalist);
+      // console.log("target", target, "source", source, "sibling", sibling);
+      el.classList.remove("Card-grey");
+      el.classList.remove("Card-blue");
+      el.classList.remove("Card-green");
+      const status = target.ariaLabel;
+      // console.log();
+      el.classList.add(
+        status == "Backlog"
+          ? "Card-grey"
+          : status == "In Progress"
+          ? "Card-blue"
+          : "Card-green"
+      );
+    });
+
+    return () => {
+      drake.destroy();
+    };
+  });
+
   return (
     <div className="Board">
       <div className="container-fluid">
diff --git a/src/app/components/Swimlane.css b/src/app/components/Swimlane.css
index 23f3880..eb478fb 100644
--- a/src/app/components/Swimlane.css
+++ b/src/app/components/Swimlane.css
@@ -3,6 +3,10 @@
     font-weight: bold;
     font-size: 18px;
   }
+
+  /* .Swimlane-Column  {
+
+  } */
   
   .Swimlane-dragColumn {
     height: calc(100vh - 100px);
diff --git a/src/app/components/Swimlane.js b/src/app/components/Swimlane.js
index 9df2aed..bc12b56 100644
--- a/src/app/components/Swimlane.js
+++ b/src/app/components/Swimlane.js
@@ -17,7 +17,7 @@ export default function Swimlane(props) {
   return (
     <div className="Swimlane-column">
       <div className="Swimlane-title">{props.name}</div>
-      <div className="Swimlane-dragColumn" ref={props.dragulaRef}>
+      <div className="Swimlane-dragColumn" ref={props.dragulaRef} aria-label={props.name}>
         {cards}
       </div>
     </div>
